Surface_mesh/include/CGAL/Surface_mesh/Surface_mesh.h

```diff
@@ -257,6 +257,7 @@ namespace CGAL {
         // increment.
         SM_Edge_index operator++(int) { SM_Edge_index tmp(*this); halfedge_ = SM_Halfedge_index((size_type)halfedge_ + 2); return tmp; }
 
+        SM_Edge_index operator+=(std::ptrdiff_t n) { halfedge_ = SM_Halfedge_index(std::ptrdiff_t(halfedge_) + 2*n); return *this; }
```

Comment:
That rises a warning in windows : https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.0-Ic-58/AABB_tree_Examples/TestReport_afabri_x64_Cygwin-Windows10_MSVC2017-Release-64bits.gz